src/components/reports/DistributionChart.vue

```diff
@@ -0,0 +1,138 @@
+<template>
```

Comment:
If this component part of a work in progress? I don't see it being imported/used anywhere.